Description: Add a missing include 
 The build fails because of missing #include <linux/udp.h>
Origin: upstream, https://github.com/relops/rtpbreak/issues/1
Bug: https://github.com/relops/rtpbreak/issues/1
Last-Update: 2016-03-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main.c
+++ b/src/main.c
@@ -32,6 +32,7 @@
 #include "common.h"
 #include "net.h"
 #include "main.h"
+#include <linux/udp.h>
 
 
 /* globals */
